integr8ly/integreatly-operator
# Get 3scale user email from identity

Set pod priority to 0 for deve sandbox installs

# Issue link
https://issues.redhat.com/browse/MGDAPI-2565
and
https://issues.redhat.com/browse/MGDAPI-2557

# What


# Verification steps - email 
Install multi-tenant RHOAM from this branch
` INSTALLATION_TYPE=multitenant-managed-api make code/run`

Install the testing-idp with "rhd" name
`NUM_REGULAR_USER=5 NUM_ADMIN=0 DEDICATED_ADMIN_PASSWORD=Password1 PASSWORD=Password1 REALM_DISPLAY_NAME="rhd" REALM="rhd" ./scripts/setup-sso-idp.sh`

Log in as test-user01 to OpenShift
Verify that the identity CR has an email test-user01@example.com

Wait for the test-user01 tenant route to be created under redhat-rhoam-3Scale
Log in using SSO
Go to Account => Users and verify that the email is `test-user01@example.com`


# Verification steps - pod priority
Check APIcast staging and Production pods and verify pod priority is set to 0
